en/engine/event/event-input.md

```diff
@@ -0,0 +1,102 @@
+# Global System Events
+
+In this section, the `global system events` of __Cocos Creator__ will be introduced.
+
+`Global system events` are irrelevant with the node hierarchy, so they are dispatched globally by `cc.systemEvent`, currently supported:
+
+- Mouse
+- Touch
+- Keyboard
+- DeviceMotion
+
+The global mouse and touch events are very similar to the node events, except that the area of action is different. The following is a description of these events.
+
+## The difference between node events and global mouse and touch events
+
+> **Note**: Before beginning this section, read up on [Auto fit for multi-resolution](../../ui-system/components/engine/multi-resolution.md#Design-resolution-and-screen-resolution) and understand the screen area and UI display area.
```

Comment:
Before beginning this section, read up on
-->
Before beginning this section, please read